features/foutafhandeling.feature

```diff
@@ -0,0 +1,292 @@
+        | code   | paramsRequired                                                                    |
+        | detail | Required request parameter 'zoek' for method parameter type String is not present |
+        En bevat de response geen invalidParams
+
+Rule: alleen gespecificeerde parameters mogen worden gebruikt
+
+    Scenario: Niet gespecificeerde parameter
+        Als '/adressen?straat=Spui' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | Bad request.                                  |
+        En bevat de response de volgende invalidParams
+        | name   | code         | reason                      |
+        | straat | unknownParam | Parameter is niet verwacht. |
+
+Rule: minimaal één optionele parameter is opgegeven
+
+    Abstract Scenario: Er zijn geen parameters opgegeven
+        Als '<path>' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                          |
+        | title  | Ten minste één parameter moet worden opgegeven. |
+        | status | 400                                             |
+        | code   | paramsRequired                                  |
+        | detail | <error detail>                                  |
+        En bevat de response geen invalidParams
+
+        Voorbeelden:
+        | path                   | error detail                                                                                                                                          |
+        | /adressen              | Precies 1 parameter van pandIdentificatie, adresseerbaarObjectIdentificatie, zoekresultaatIdentificatie, postcode of huisnummer moet worden opgegeven |
+        | /adresseerbareobjecten | Precies 1 parameter van nummeraanduidingIdentificatie, pandIdentificatie, pandIdentificaties of bbox moet worden opgegeven                            |
+        | /panden                | Precies 1 parameter van adresseerbaarObjectIdentificatie, nummeraanduidingIdentificatie, locatie of bbox moet worden opgegeven                        |
+
+Rule: opgegeven parameter(s) heeft een waarde
+
+    Scenario: Er is geen waarde voor één parameter opgegeven
+        Als '/adressen?pandIdentificatie=' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                                                                                                                |
+        | title  | Ten minste één parameter moet worden opgegeven.                                                                                                       |
```

Comment:
ik vind dit een vreemde foutmelding. Er is wel een pandIdentificatie opgegeven, maar die voldoet niet aan pattern ^[0-9]{16}$

Reply:
Dit is zoals het nu werkt. Ik had ook graag een andere melding gehad. Maar dit is wel wat je krijgt bij een generieke foutafhandeling specificatie